handshake_messages.go

```diff
@@ -1276,10 +1276,12 @@ func (m *newSessionTicketMsgTLS13) unmarshal(data []byte) bool {
 }
 
 type certificateRequestMsgTLS13 struct {
+	original                         []byte
```

Comment:
In non u_\* files let's comment changes to distinguish between utls and stdlib code, it makes it easier when merging with upstream. See [here](https://github.com/refraction-networking/utls/blob/c567269913b5e46f670ec8eeebc16a6ab9fc41d4/handshake_client_tls13.go#L754-L757) and [here](https://github.com/refraction-networking/utls/blob/c567269913b5e46f670ec8eeebc16a6ab9fc41d4/handshake_client_tls13.go#L46)